box/config.yml

```diff
@@ -45,49 +48,32 @@ nodejs_install_npm_user: "{{ drupalvm_user }}"
 npm_config_prefix: "/home/{{ drupalvm_user }}/.npm-global"
 installed_extras:
   - adminer
+  - drush
+  - mailhog
   - nodejs
   - selenium
-  - memcached
+  - xdebug
 
-# Use PHP 5.6 and change package names to work with 5.6.
+# Use PHP 5.6.
 php_version: "5.6"
-php_install_recommends: no
-php_packages:
-  - php5.6
-  - php5.6-apcu
-  - php5.6-mcrypt
-  - php5.6-cli
-  - php5.6-common
-  - php5.6-curl
-  - php5.6-dev
-  - php5.6-fpm
-  - php5.6-gd
-  - php5.6-sqlite3
-  - php5.6-xml
-  - php5.6-mbstring
-  - php-pear
-  - libpcre3-dev
-  - php5.6-bz2
-php_conf_paths:
-  - /etc/php/5.6/fpm
-  - /etc/php/5.6/apache2
-  - /etc/php/5.6/cli
-php_extension_conf_paths:
-  - /etc/php/5.6/fpm/conf.d
-  - /etc/php/5.6/apache2/conf.d
-  - /etc/php/5.6/cli/conf.d
-php_fpm_daemon: php5.6-fpm
-php_fpm_conf_path: "/etc/php/5.6/fpm"
-php_fpm_pool_conf_path: "/etc/php/5.6/fpm/pool.d/www.conf"
-php_mysql_package: php5.6-mysql
-php_redis_package: php5.6-redis
-php_memcached_package: php5.6-memcached
+php_packages_extra:
```

Comment:
@mikemadison13 Where'd you find `php_packages_extra` and does it work? I just updated my work project to the latest DrupalVM and only found this (much shorter than before) [Drupal VM PHP 5.6 documentation page](http://docs.drupalvm.com/en/latest/configurations/php/) which recommended adding php packages to the `extra_packages` list.